ACKNOWLEDGEMENTS.md

```diff
@@ -0,0 +1,8 @@
+This project is managed by Prof. Justin Cappos and other members of the Secure
+Systems Lab at NYU.
+
+This material is based upon work supported by the National Science Foundation
+under Grant Nos. CNS-1345049 and CNS-0959138. Any opinions, findings, and
```

Comment:
Should this still be here as an example or should we remove these so that others fill them in?

Reply:
This is meant to be a more general comment, whether we should have this more as a template or an example.  I can see pros and cons to both.

Reply:
I guess these are more examples than template files.  Perhaps we can edit the wording in the README to make this clear.  What do you think?

Some of the other "templates" suffer from the same shortcoming.  For example, the travis configuration file would need to be edited, because not all projects use the same branch names, test suite, etc.

I hope that other projects can use these examples as a starting point.

Reply:
The README was revised to use "sample documents" instead of "templates".  Please have a look at the new README and let me know if it works better.  If so, I can go ahead and rename the repository, as well.

Reply:
I think perhaps also making some edits in the files to show where to change things is wise.  (If you disagree, let's discuss.)